packages/console/src/pages/Dashboard/index.tsx

```diff
@@ -23,35 +33,102 @@ const Dashboard = () => {
         <div className={styles.subtitle}>{t('dashboard.description')}</div>
       </div>
       {!isLoading && (
-        <div className={styles.topBlocks}>
-          <Card className={styles.block}>
-            <div className={styles.title}>{t('dashboard.total_users')}</div>
-            <div className={styles.content}>
-              <div className={styles.number}>{totalData.totalUserCount}</div>
-            </div>
-          </Card>
-          <Card className={styles.block}>
-            <div className={styles.title}>{t('dashboard.new_users_today')}</div>
-            <div className={styles.content}>
-              <div className={styles.number}>{newData.today.count}</div>
-              <div className={classNames(styles.delta, newData.today.delta < 0 && styles.down)}>
-                ({newData.today.delta > 0 && '+'}
-                {newData.today.delta}){newData.today.delta > 0 ? <ArrowUp /> : <ArrowDown />}
+        <>
+          <div className={styles.blocks}>
+            <Card className={styles.block}>
+              <div className={styles.title}>{t('dashboard.total_users')}</div>
+              <div className={styles.content}>
+                <div className={styles.number}>{totalData.totalUserCount}</div>
               </div>
-            </div>
-          </Card>
-          <Card className={styles.block}>
-            <div className={styles.title}>{t('dashboard.new_users_7_days')}</div>
-            <div className={styles.content}>
-              <div className={styles.number}>{newData.last7Days.count}</div>
-              <div className={classNames(styles.delta, newData.last7Days.delta < 0 && styles.down)}>
-                ({newData.last7Days.delta > 0 && '+'}
-                {newData.last7Days.delta})
-                {newData.last7Days.delta > 0 ? <ArrowUp /> : <ArrowDown />}
+            </Card>
+            <Card className={styles.block}>
+              <div className={styles.title}>{t('dashboard.new_users_today')}</div>
+              <div className={styles.content}>
+                <div className={styles.number}>{newData.today.count}</div>
+                <div className={classNames(styles.delta, newData.today.delta < 0 && styles.down)}>
+                  ({newData.today.delta > 0 && '+'}
+                  {newData.today.delta}){newData.today.delta > 0 ? <ArrowUp /> : <ArrowDown />}
```

Comment:
`newData.today.delta` is used multiple times here. Should extract a const

Reply:
refactored